4 language selector

ujjwal5 opened the pull request:
Please always provide the [GitHub issue(s)](../issues) your PR is for, as well as test URLs where your change can be observed (before and after):

Fix <https://github.com/hlxsites/fonterra-fernleaf/issues/4>

Test URLs:
- Before: https://main--fonterra-fernleaf--hlxsites.hlx.page/
- After: https://4-language-selector--fonterra-fernleaf--hlxsites.hlx.page/en

mkbansal1 reviewed (changes requested):
@ujjwal5 Please fix below issues

- Scrollbar to be removed
<img width="1034" alt="image" src="https://github.com/hlxsites/fonterra-fernleaf/assets/129100763/e3afad01-3b59-42f6-9056-ba27fee50b55">

- Clicking outside popup should close the popup
- Fix lighthouse issue
<img width="1014" alt="image" src="https://github.com/hlxsites/fonterra-fernleaf/assets/129100763/f1a5bff5-a6e1-433b-b397-63f4a856dd18">

ujjwal5 commented:
> @ujjwal5 Please fix below issues
> 
> * Scrollbar to be removed
> 
> <img alt="image" width="1034" src="https://user-images.githubusercontent.com/129100763/257990058-e3afad01-3b59-42f6-9056-ba27fee50b55.png">
> * Clicking outside popup should close the popup
> * Fix lighthouse issue
> 
> <img alt="image" width="1014" src="https://user-images.githubusercontent.com/129100763/257990313-f1a5bff5-a6e1-433b-b397-63f4a856dd18.png">

Addressed all your comments @mkbansal1

mkbansal1 commented:
@ujjwal5 Last point from my side, flag icon image used is too big
https://github.com/hlxsites/fonterra-fernleaf/blob/621c071ee9903558dd2cc875df8bcb79dfc81afb/styles/images/flag-malaysia.png
<img width="358" alt="image" src="https://github.com/hlxsites/fonterra-fernleaf/assets/129100763/5eb3b246-f48b-4568-af74-6e3b667e1fde">
Use a correct size image, max 48x48
